app-catalog/src/components/charts/List.tsx

```diff
@@ -16,15 +17,25 @@ import {
   TextField,
   Tooltip,
   Typography,
+  useTheme,
 } from '@mui/material';
 import { Autocomplete, Pagination } from '@mui/material';
 import { useEffect, useState } from 'react';
 //import { jsonToYAML, yamlToJSON } from '../../helpers';
 import { fetchChartsFromArtifact } from '../../api/charts';
-import CNCFLight from './cncf-icon-color.svg';
 //import { createRelease } from '../../api/releases';
 import { EditorDialog } from './EditorDialog';
 
+interface AppCatalogConfig {
+  /**
+   * Show only verified packages. If set to false shows all the packages
+   */
+  showVerified: boolean;
```

Comment:
`showVerified` isn't accurate as a name. Really it is `showOnlyVerified`. Can you please change it?

Reply:
done!